cloud/pkg/devicecontroller/controller/downstream.go

```diff
@@ -308,9 +310,22 @@ func (dc *DownstreamController) sendDeviceModelMsg(device *v1beta1.Device, opera
 	if device == nil || device.Spec.DeviceModelRef == nil {
 		return
 	}
-	edgeDeviceModel, ok := dc.deviceModelManager.DeviceModel.Load(device.Spec.DeviceModelRef.Name)
-	if !ok {
-		klog.Warningf("not found device model for device: %s, operation: %s", device.Name, operation)
+	var edgeDeviceModel any
+	var ok bool
+	err := retry.Do(
+		func() error {
+			edgeDeviceModel, ok = dc.deviceModelManager.DeviceModel.Load(device.Spec.DeviceModelRef.Name)
+			if !ok {
+				return fmt.Errorf("not found device model for device: %s, operation: %s", device.Name, operation)
+			}
+			return nil
+		},
+		retry.Delay(1*time.Second),
+		retry.Attempts(10),
```

Comment:
Why set 10 retry times?

Reply:
This 10 attempts is only to ensure multiple retries after a failed attempt, without any specific requirements.